pyface/i_image_resource.py

```diff
@@ -35,68 +37,6 @@ class IImageResource(Interface):
     #: for the image (see the resource manager for more details).
     search_path = List()
 
-    # ------------------------------------------------------------------------
-    # 'object' interface.
-    # ------------------------------------------------------------------------
-
-    def __init__(self, name, search_path=None):
-        """ Creates a new image resource. """
-
-    # ------------------------------------------------------------------------
-    # 'ImageResource' interface.
-    # ------------------------------------------------------------------------
-
-    def create_image(self, size=None):
-        """ Creates a toolkit specific image for this resource.
-
-        Parameters
-        ----------
-        size : (int, int) or None
-            The desired size as a width, height tuple, or None if wanting
-            default image size.
-
-        Returns
-        -------
-        image : toolkit image
-            The toolkit image corresponding to the resource and the specified
-            size.
-        """
-
-    # FIXME v3: The need to distinguish between bitmaps and images is toolkit
-    # specific so, strictly speaking, the conversion to a bitmap should be done
-    # wherever the toolkit actually needs it.
```

Comment:
Is there any useful information in this fixme that we want to retain? _i don't think so IIUC_

Reply:
No - this came from Wx wanting `wx.Bitmap` objects in many cases, where Qt in the same situation likes `QImage` objects (or sometimes vice-versa).  Part of this change is making the distinction more concrete, but toolkits won't change what they actually use.